xlsynth-driver/tests/invoke_test.rs

```diff
@@ -2940,3 +2940,146 @@ fn simulate_basic_valid_pipeline(
     .expect("simulation succeeds");
     assert!(vcd.contains("$var"));
 }
+
+const QUICKCHECK_DSLX: &'static str = r#"
+fn f(x: u8) -> bool { x == x }
+#[quickcheck] fn qc_success(x: u8) -> bool { f(x) }
+#[quickcheck] fn qc_failure(x: u8) -> bool { x == u8:0 }
+"#;
+
+macro_rules! test_prove_quickcheck_solver {
```

Comment:
I feel like this would also be nice to have as common subroutines that get called from test entry points instead of a macro -- remember you can use test_case() to parameterize the test entry points.